<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#frmMain{
	background:white;
}</string>
  </property>
  <widget class="QStackedWidget" name="Bottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>1111</height>
    </rect>
   </property>
   <widget class="QWidget" name="Main">
    <widget class="QPushButton" name="SystemOptimization">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>990</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:8px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string> 一 键 优 化</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>850</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UISreachIP">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>客户端界面</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="TempGroupBox">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>420</y>
       <width>111</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#TempGroupBox
{
	border-radius:8px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="title">
      <string>s's's's</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Server">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器端</string>
     </property>
    </widget>
    <widget class="QLabel" name="SystemScore">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>671</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>420</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#282A49</string>
     </property>
     <property name="text">
      <string>47</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>850</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color:#282A49</string>
     </property>
     <property name="text">
      <string>系统健康评分</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="HumidityGroupBox">
     <property name="geometry">
      <rect>
       <x>1460</x>
       <y>180</y>
       <width>111</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#HumidityGroupBox
{
	border-radius:8px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="title">
      <string>s's's's</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Pm25GroupBox">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>120</y>
       <width>111</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#Pm25GroupBox
{
	border-radius:8px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="title">
      <string>s's's's</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HouseIntelligence">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>990</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#HouseIntelligence
{
	border-radius:8px;
	background:#FFFFFF
}</string>
     </property>
     <property name="text">
      <string> 全 屋 智 能</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/C:/Users/Alumer/Desktop/AI (2).png</normaloff>:/C:/Users/Alumer/Desktop/AI (2).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="LuxGroupBox">
     <property name="geometry">
      <rect>
       <x>1710</x>
       <y>410</y>
       <width>111</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#LuxGroupBox
{
	border-radius:8px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="title">
      <string>s's's's</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>910</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#282A49</string>
     </property>
     <property name="text">
      <string>评价系统当前的耗能与安全</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ControlPage">
    <widget class="QGroupBox" name="group">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>160</y>
       <width>741</width>
       <height>851</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>800</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>当前时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>770</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>Integrated space
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>650</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string>color:#282A49</string>
      </property>
      <property name="text">
       <string>系统健康评分</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>710</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>评价系统当前的耗能与安全</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>800</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>2022.3.17</string>
      </property>
     </widget>
     <widget class="QLabel" name="SystemScore_ControlPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>761</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>500</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#282A49</string>
      </property>
      <property name="text">
       <string>47</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>1081</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#groupBox{
	border:none;
	background-color:#282A49;
	border-radius:15px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="GoMain">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>220</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;
border-radius:50px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>Main</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GoControlPage">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>650</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;
border-radius:50px;
background-color: white;
 border-style: solid;</string>
      </property>
      <property name="text">
       <string>Control</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ButtonBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>731</width>
       <height>1001</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#ButtonBox{
	border:none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="DoorSystem">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="LightSystem">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>320</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="AIControlPage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>840</y>
        <width>621</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="AirSystem">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>590</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="WindowSystem">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>590</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="WindSystem">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="CurtainSystem">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>251</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</normaloff>:/C:/Users/Alumer/Desktop/SystemOp.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
